impala/_thrift_api.py

```diff
@@ -63,7 +63,7 @@
 if six.PY3:
     # import thriftpy2 code
     from thriftpy2 import load
```

Comment:
Should we always be using thriftpy2 instead of the thrift package in Python 3? Maybe leave a comment to that effect?

Reply:
Yep that would work